packages/now-cli/src/commands/domains/ls.ts

```diff
@@ -84,28 +112,92 @@ export default async function ls(
   return 0;
 }
 
-function formatDomainsTable(domains: Domain[]) {
-  const current = new Date();
-  return table(
-    [
-      [
-        '',
-        chalk.gray('domain'),
-        chalk.gray('serviceType'),
-        chalk.gray('verified'),
-        chalk.gray('cdn'),
-        chalk.gray('age'),
-      ].map(s => chalk.dim(s)),
-      ...domains.map(domain => {
-        const url = chalk.bold(domain.name);
-        const time = chalk.gray(ms(current.getTime() - domain.createdAt));
-        return ['', url, domain.serviceType, domain.verified, true, time];
-      }),
-    ],
-    {
-      align: ['l', 'l', 'l', 'l', 'l'],
-      hsep: ' '.repeat(4),
-      stringLength: strlen,
+function createDomainsInfo(domains: Domain[], projects: Project[]) {
+  const info = new Map<string, DomainInfo>();
+
+  domains.forEach(domain => {
+    info.set(domain.name, {
+      domain: domain.name,
+      apexDomain: domain.name,
+      projectName: null,
+      expiresAt: domain.expiresAt || null,
+      createdAt: domain.createdAt,
+      configured: Boolean(domain.verified),
+      dns: domain.serviceType === 'zeit.world' ? 'Vercel' : 'External',
```

Comment:
Don't we have a new domain for vercel here?

Reply:
@mglagola do you know?

Reply:
We are still using `zeit.world` to suggest a domain is using Vercel nameservers